python/setup.py

```diff
@@ -252,6 +286,20 @@ def config_cython():
                         ),
                     )
                 )
+                data_files.append(
+                    (
+                        "dgl/graphbolt",
+                        glob.glob(
+                            os.path.join(
+                                os.path.dirname(
+                                    os.path.relpath(path, CURRENT_DIR)
+                                ),
+                                "graphbolt",
+                                BOLT_LIB_PATTERN,
+                            )
+                        ),
+                    )
+                )
```

Comment:
Similarly, we could refactor the code to make it more reusable.

Reply:
Ditto